html/rendering/replaced-elements/attributes-for-embedded-content-and-images/img-aspect-ratio.html

```diff
@@ -95,6 +104,8 @@
 
   test(function () {
     assert_not_equals(images[5].offsetHeight, 500, "Images with alt text should be inline and ignore the aspect ratio");
+    // Though aspect-ratio is ignored, its value does not change.
+    assert_equals(getComputedStyle(images[5]).aspectRatio, "auto 100 / 500");
```

Comment:
This test previously passed in Chrome and Safari but will now fail. Are there bugs filed for whatever the problem here is?

Reply:
Not yet. This corner case is found when I tried to implement the CSS property approach for WebKit. https://bugs.webkit.org/show_bug.cgi?id=217529
The behavior of Chromium and Fixfox are not consistency. After discussing with Christian, we agreed that the value shouldn't be changed.
So I'm trying to add a case for it.
For WebKit, this should pass after the patch in https://bugs.webkit.org/show_bug.cgi?id=217529
For Chromium, I think I can file a bug for it, maybe after landing this patch?

Reply:
Filed a bug:) https://bugs.chromium.org/p/chromium/issues/detail?id=1214807

Reply:
Thanks!